packages/rs-dpp/src/data_contract/state_transition/data_contract_update_transition/mod.rs

```diff
@@ -57,8 +57,8 @@ pub struct DataContractUpdateTransition {
     pub protocol_version: u32,
     #[serde(rename = "type")]
     pub transition_type: StateTransitionType,
-    // we want to skip serialization of transitions, as we does it manually in `to_object()`  and `to_json()`
-    #[serde(skip_serializing)]
+    // we want to skip serialization of transitions, as we do it manually in `to_object()`  and `to_json()`
+    // #[serde(skip_serializing)]
```

Comment:
Just comment?

Reply:
I'm not sure what to do about this one - this one was done for a reason, but it broke stuff. So while we're not 100% sure (it seems like it doesn't) it doesn't break anything I would keep it here

Reply:
Let's bring it back then. My point is that we shouldn't commit commented code.